src/docs/product/alerts/create-alerts/crash-rate-alert-config.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: Crash-rate Alert Configuration
+sidebar_order: 20
+redirect_from:
+ - /product/sentry-basics/guides/alert-notifications/crash-rate-alerts/
+description: "Learn more about the options for configuring a crash rate alert."
+---
+
+Sentry provides several configuration options to create a crash rate alert based on your organization's needs.
+
+### Environment
+
+Specify which <PlatformLink to="/configuration/environments/">environment(s)</PlatformLink> will use this particular alert rule. This control filters on the `environment` tag in your events. This filter is helpful because the urgency and workflows you apply to production alerts might differ from those you apply to alerts originating from your QA environment, for example.
+
+The “Env:” dropdown list here has the same environments that are available for the selected project in the global “Environment” dropdown (this does not include hidden environments). Selecting "All" is equivalent to having no environment filter.
+
+### Tags & Attributes
+
+Add filters in the field provided to narrow down what you'll be alerted about. Crash rate alert support release tag to narrow down your alert to a specific release.
+
+## Crash Rate Alerts (Function + Time Interval)
+
+Depending on the type of crash rate alert you’ve selected, the corresponding function is built into the alert and the settings aren't displayed. For example, if you select “Crash Free Sessions Rate”, that translates to the function, `percentage(sessions_crashed, sessions)`. Since editing this function would change the nature of the alert, it is not editable and thus hidden.
```

Comment:
```suggestion
Depending on the type of crash-rate alert you’ve selected, the corresponding function is built into the alert and the settings aren't displayed. For example, if you select “Crash Free Sessions Rate”, that translates to the function, `percentage(sessions_crashed, sessions)`. Since editing this function would change the nature of the alert, it is not editable and thus hidden.
```